windows/start.sh

```diff
@@ -102,7 +102,12 @@ echo
 #cd #Bad: working dir should be whatever directory was invoked from rather than fixed to the Home folder
 
 docker () {
-  MSYS_NO_PATHCONV=1 docker.exe "$@"
+  args=()
+  for arg in "$@"
+  do
+    args+="$(echo $arg | sed -e 's/type=bind,\(source\|src\)=\([A-Za-z]\):\\\\/type=bind,\1=\/\L\2\//' | sed -e 's/\([[:print:]]\)\\/\1\//') "
```

Comment:
These `sed` commands run unconditionally, the second one being a problem: it converts all `<x>\` to `<x>/`.  It should be guarded with the `^type=bind,...` bits too; I can't think of a more elegant way at the moment.  Also, the `^` is missing from the first `sed` command (should be `s/^type=bind,...`).